packages/shared/lib/profile.ts

```diff
@@ -450,3 +451,9 @@ export const validateProfileName = (trimmedName: string): void => {
         throw new Error(locale('error.profile.duplicate'))
     }
 }
+
+selectedAccountId.subscribe((accountId) => {
```

Comment:
Code looks good and works, but let's group this more to the top (under the subscription for profiles). Now it's hidden under validateProfileName and it will be tough to find this one if we would need to change it in the future.